configs/eth_odin_v4.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
the generic ethernet config has the button item enabled, and I tried to follow suit with the other board-specific deviations in the set of configurations.